src/test/java/com/thomashan/coup/game/StandardGameScenarioTest.java

```diff
@@ -0,0 +1,134 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StandardGameScenarioTest {
+    @Test
+    public void test_AssassinateChallenger_Challenge_ChallengeSuccess() {
+        Game game = SetupGame.create(
+                Player.of(CAPTAIN, AMBASSADOR).income(),
+                Player.of(CAPTAIN, AMBASSADOR));
+
+        game = game.action(MainAction.of(game.getCurrentPlayer(), ASSASSINATE, game.getPlayers().others(game.getCurrentPlayer()).get(0)));
+        game = game.action(ChallengeAction.of(game.getPlayers().others(game.getCurrentPlayer()).get(0), CHALLENGE));
+        game = game.action(RevealCardAction.of(game.getCurrentPlayer(), AMBASSADOR));
+
+        assertEquals(1, game.getTurn().getTurnNumber());
+        assertEquals(0, game.getPlayers().get(0).getCoins());
+        assertEquals(1, getOtherPlayer(game).getActiveCardSet().size());
+    }
+
+    @Test
+    public void test_AssassinateChallenger_Challenge_ChallengeFail_ShouldMakeChallengerInactive() {
+        Game game = SetupGame.create(
+                Player.of(ASSASSIN, AMBASSADOR).income(),
+                Player.of(CAPTAIN, AMBASSADOR));
+
+        game = game.action(MainAction.of(game.getCurrentPlayer(), ASSASSINATE, getOtherPlayer(game)));
+        game = game.action(ChallengeAction.of(getOtherPlayer(game), CHALLENGE));
+        game = game.action(RevealCardAction.of(getOtherPlayer(game), AMBASSADOR));
+        game = game.action(BlockAction.noBlock(getOtherPlayer(game)));
+        game = game.action(RevealCardAction.of(getOtherPlayer(game), CAPTAIN));
+
+        assertEquals(1, game.getTurn().getTurnNumber());
+        assertEquals(0, game.getPlayers().get(0).getCoins());
+        assertTrue(game.isComplete());
+    }
+
+    @Test
+    public void test_AssassinateChallenger_Challenge_ChallengeFail_ChallengerBlock_ChallengeBlock_ShouldMakeChallengerInactive() {
+        Game game = SetupGame.create(
+                Player.of(ASSASSIN, AMBASSADOR).income(),
+                Player.of(CAPTAIN, AMBASSADOR));
+
+        game = game.action(MainAction.of(game.getCurrentPlayer(), ASSASSINATE, getOtherPlayer(game)));
+        game = game.action(ChallengeAction.of(getOtherPlayer(game), CHALLENGE));
+        game = game.action(RevealCardAction.of(getOtherPlayer(game), AMBASSADOR));
+        game = game.action(BlockAction.block(getOtherPlayer(game), BLOCK_ASSASSINATE, CONTESSA));
+        game = game.action(ChallengeAction.of(game.getCurrentPlayer(), CHALLENGE));
+        game = game.action(RevealCardAction.of(getOtherPlayer(game), CAPTAIN));
+
+        assertEquals(1, game.getTurn().getTurnNumber());
+        assertEquals(0, game.getPlayers().get(0).getCoins());
+        assertTrue(game.isComplete());
+    }
+
+    @Test
+    public void test_AssassinateChallenger_Allow_ChallengerBlock_AllowBlock_ShouldGoToNextTurn() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unit tests should not contain more than 1 assert(s).](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)